bitmaps.h

```diff
@@ -0,0 +1,211 @@
+#pragma once
+
+#include <string>
+#include <set>
+#include <vector>
+#include <array>
+
+#include "context.h"
+#include "common.h"
+#include "resourcew.h"
+
+constexpr static unsigned int BITMAP_WH = 128;
+
+class BitmapData : public Identifiable<std::array<GLubyte, BITMAP_WH * BITMAP_WH>> {
+public:
+	BitmapData(const std::initializer_list<GLubyte> &i_list);
+	BitmapData(const GLubyte *data);
+};
+
+enum class BitmapGroup {
+	Yokin,
+	Impostor,
+	YoyImpostor,
+};
+
+struct Bitmaps {
+	struct Definition {
+		auto operator<=>(const Definition &other) const = default;
+
+		std::wstring name;
```

Comment:
Sadly I cannot make these fields `const`, because if I want this type to be copy-assignable, which is necessary to use in some `<algorithm>` stuff, they cannot be `const`.